src/nvim/version.c

```diff
@@ -79,50 +176,50 @@ static int included_patches[] = {
   // 1048,
   // 1047,
   // 1046,
-  // 1045,
-  // 1044,
-  // 1043,
+  // 1045 NA,
+  // 1044 NA,
+  // 1043 NA,
   // 1042,
   // 1041,
-  // 1040,
+  // 1040 NA,
   // 1039,
-  // 1038,
+  // 1038 NA,
   // 1037,
   // 1036,
   // 1035,
   // 1034,
-  // 1033,
+  // 1033 NA,
   1032,
-  // 1031,
+  // 1031 NA,
   // 1030,
   // 1029,
   // 1028,
-  // 1027,
-  // 1026,
-  // 1025,
-  // 1024,
-  // 1023,
-  // 1022,
-  // 1021,
-  // 1020,
-  // 1019,
+  // 1027 NA,
```

Comment:
Is not mentioned in commit message. Perhaps `vim_str2nr()` parameter change could be useful.